Features/CaseSearch/test_cases/test_04_casesearch_fuzzy_search.py

```diff
@@ -102,17 +105,16 @@ def test_case_05_remove_special_characters(driver):
                                         expected_value=CaseSearchUserInput.id_without_hyphen)
 
 
-# Elastic search takes time to reflect the case so executing this last
 def test_case_06_claimed_cases_on_report(driver, settings):
     report = HomePage(driver, settings)
     load = ReportPage(driver)
     user = MobileWorkerPage(driver)
     casesearch = CaseSearchWorkflows(driver)
     report.reports_menu()
     load.case_list_report()
-    time.sleep(200)  # explore case list explorer
+    time.sleep(90)  # Elastic search takes time to reflect the case
     casesearch.check_case_claim_case_type(claimed_case_name=CaseSearchUserInput.song_case_bugs_user2,
                                           claimed_user=user.username + "@casesearch.commcarehq.org")
     """Delete the user"""
-    user.mobile_worker_menu()
+    driver.get("https://staging.commcarehq.org/a/casesearch/settings/users/commcare/")
```

Comment:
Try not to send this url as hard coded in the testcase files. Either add the url in user input or add the step in test pages file.

Reply:
Moved in https://github.com/dimagi/dimagi-qa/pull/184/commits/5741c2a5600a382b11c98d3c99d4d335da5e56e4

P.S. the UI click is failing for me and I don't want to spend a ton of time in debugging it.